src/main/java/Duke.java

```diff
@@ -6,5 +8,82 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        String dashes = "____________________________________________________________";
+
+        String intro = "Hello! I'm Duke\nWhat can I do for you?";
+        System.out.println(intro);
+        System.out.println(dashes);
+        Task[] List = new Task[100];
+        String[] List2 = new String[100];
+        int Count = 0;
```

Comment:
Variables should be in camel case, could consider changing the variable naming to follow the convention.